lib/generators/graphiti/install_generator.rb

```diff
@@ -46,20 +61,13 @@ def install
       end
 
       insert_into_file "config/routes.rb", after: "Rails.application.routes.draw do\n" do
-        if defined?(VandalUi)
           <<-STR
-  scope path: ApplicationResource.endpoint_namespace, defaults: { format: :jsonapi } do
-    mount VandalUi::Engine, at: '/vandal'
-    # your routes go here
+  scope path: ApplicationResource.endpoint_namespace, defaults: { format: :jsonapi }\
```

Comment:
I think you could use the [squiggly heredoc](https://stackoverflow.com/a/34888238) to ignore leading whitespace if you wanted to indent this a bit.